### PR TITLE
feat(carousel): adding actionable items into carousel 

### PR DESCRIPTION
**Pull request by kolaps33**
Adding actionable items into the carousel. 
Trying to add button, buttons and toolbar to carousel slide. 

If user is inside the carousel item and press right/left arrow key, then rotation of carousel was still working. This is fix with stopPropagation.

BE CAREFUL carousel is not fully accessible, related bugs reported in the ticket:
https://github.com/microsoft/fluent-ui-react/issues/2272

### CODE REVIEW

File: packages/accessibility/src/behaviors/Carousel/carouselItemBehavior.ts [hunk 1]
```diff
@@ -16,8 +17,13 @@ const carouselItemBehavior: Accessibility<CarouselItemProps> = props => ({
   },
 
   keyActions: {
-    root: {},
-  },
+    root: {
+      stopPropagation: {
```
Comment (mnajdova) [inline packages/accessibility/src/behaviors/Carousel/carouselItemBehavior.ts]: This looks like a strange name for keyAction. Can we use something related to the fact that navigation happend?

File: packages/accessibility/src/behaviors/Carousel/carouselItemBehavior.ts [hunk 2]
```diff
@@ -16,7 +17,11 @@ const carouselItemBehavior: Accessibility<CarouselItemProps> = props => ({
   },
 
   keyActions: {
-    root: {},
+    root: {
+      arrowKeysNavigationStopPropagation: {
```
Comment (silviuaavram) [inline packages/accessibility/src/behaviors/Carousel/carouselItemBehavior.ts:22]: Let's add a unit test for this.

It should check that left/right arrows sent to a child of the container (the visible item for instance, or a child of that visible item) should not move the slides.

**Review by silviuaavram — COMMENTED** [review]
Done :)

File: packages/accessibility/src/behaviors/Carousel/carouselItemBehavior.ts (hunk 2 above)
Comment (kolaps33) [reply packages/accessibility/src/behaviors/Carousel/carouselItemBehavior.ts:22]: thanks :) for this catch. Tests added,  as well a specification line in the top of file.

File: packages/react/test/specs/components/Carousel/Carousel-test.tsx [hunk 3]
```diff
@@ -150,6 +159,26 @@ describe('Carousel', () => {
 
       expect(pagination.getDOMNode().textContent).toBe(`1 of ${items.length}`)
     })
+
+    it('should not change at arrow left if event is invoked on child element', () => {
```
Comment (silviuaavram) [inline packages/react/test/specs/components/Carousel/Carousel-test.tsx:163]: either make starting slide greater than 1 or make it circular, so you make sure that it has the possibility to go to the left.

**Review by silviuaavram — APPROVED** [review]
address the last comment. otherwise lgtm!

File: packages/react/test/specs/components/Carousel/Carousel-test.tsx (hunk 3 above)
Comment (kolaps33) [reply packages/react/test/specs/components/Carousel/Carousel-test.tsx:163]: thanks 👍  
added circular prop :)

**Pull request merged**